### PR TITLE
button sounds

### PR DESCRIPTION
**Pull request by thomas11111111**
clippy clicky high pitched

### CODE REVIEW

**Comment by sprunk** [comment]
sources into origin.txt

File: sounds/origin.txt [hunk 1]
```diff
@@ -27,6 +27,10 @@ entire announcer folder contents
 Kmar:
 entire announcer/kmar folder contents
 
+Thomas1:
+click-full.wav      (zapsplat_multimedia_button_click_solid_hard_001_49162)
```
Comment (sprunk) [inline sounds/origin.txt:31]: What does "zapsplat_multimedia_button_click_solid_hard_001_49162" mean? How do I use this name.

**Comment by GoogleFrog** [comment]
What are these for?

**Comment by thomas11111111** [comment]
They are suited for navigating big menu tabs/battle list (full)  and small sub tabs like the chat tabs or settings sub tabs (empty).
But could be for anything, game startup, game exit, register, login/out, hyperlink, playing/spectating, back/leave, game starting, matchmaking queue joined, matchmaking accept timed out,

**Comment by GoogleFrog** [comment]
Then they should go in the zkmenu repository. This is the game proper repository.

**Pull request closed without merging**